server/utils.go

```diff
@@ -92,11 +92,15 @@ func parseJIRAIssuesFromText(text string, keys []string) []string {
 	return issues
 }
 
-func getIssueURL(i *JIRAWebhookIssue) string {
-	u, _ := url.Parse(i.Self)
-	return u.Scheme + "://" + u.Host + "/browse/" + i.Key
+func getIssueURL(issue *jira.Issue) string {
+	if issue == nil {
+		return ""
+	}
+	u, _ := url.Parse(issue.Self)
+	return u.Scheme + "://" + u.Host + "/browse/" + issue.Key
 }
 
-func getUserURL(issue *JIRAWebhookIssue, user *jira.User) string {
+func getUserURL(user *jira.User) string {
+	// TODO is this right?
```

Comment:
This function is not used in 2.0, and will likely be deprecated with @cpoile's implementation of `/jira settings notify`. 

I can cut a separate PR and simply remove all "notify" functionality from v2.0 since it's not used. This function will disappear then.